x-pack/docs/en/security/configuring-stack-security.asciidoc

```diff
@@ -58,32 +25,53 @@ this {es} cluster.
 Keystore that contains the key and certificate for the HTTP layer for this node.
 
 `transport_keystore_all_nodes.p12`::
-Keystore that contains the key and certificate for the transport layer for all the nodes in your cluster.
---
+Keystore that contains the key and certificate for the transport layer for all
+the nodes in your cluster.
+
+You can then start {kib} and enter the enrollment token, which is valid for 30
+minutes. This token automatically applies the security settings from your {es} 
+cluster, authenticates to {es} with the `kibana_system` user, and writes the 
+security configuration to `kibana.yml`. 
+
+A security certificate is generated in the {kib} `/data` directory. This
```

Comment:
Same comment here about this being an fyi-only non-actionable piece of info. Not sure if we have a way to mark it as is or if it makes sense to do so.

Reply:
This is  a CA certificate and it is not generated, but rather retrieved from elasticsearch.

Reply:
I'll clarify where this file comes from and then add it to the new section at the bottom of the page with the other files and descriptions.